OWML.ModLoader/ModFinder.cs

```diff
@@ -53,17 +53,12 @@ private IModData GetModData(IModManifest manifest)
             }
             else if (defaultConfig != null)
             {
-                foreach (var setting in defaultConfig.Settings)
-                {
-                    if (!config.Settings.ContainsKey(setting.Key))
-                    {
-                        config.Settings.Add(setting.Key, setting.Value);
-                    }
-                }
+                config.MakeConfigConsistentWithDefaults(defaultConfig);
             }
+
+
```

Comment:
double newline? why?

Reply:
good catch, would have thought that VS would auto-fix since I have it configured for that